### ISSUE TITLE
Update dep to support Poison 3.x

### ISSUE THREAD

**Issue by karlseguin** [rubencaro/cipher#11]
Currently get:
```
Failed to use "poison" (version 3.1.0) because
  cipher (version 1.3.0) requires ~> 2.0
```

People can get passed this by including poison with `override: true`:

```
{:poison, "~> 3.1", override: true},
```

But would be nice to have it handled in cipher

**Comment by rubencaro** [comment]
Fixed in #10

**Comment by rubencaro** [comment]
Included in release 1.3.1
